docs/source/ephys_at_swc/example_pipelines/examples/sara_mederos.py

```diff
@@ -0,0 +1,110 @@
+"""
+NP2.0 in SpikeInterface
+=======================
+
+Sara Mederos (Hofer Lab) performs chronic electrophysiological
+recordings from subcortical and cortical areas using 4-shank
+Neuropixels 2.0 probes (Open Ephys). Recordings are conducted
```

Comment:
Just to make it super obvious what this means to anyone new to the field
```suggestion
Neuropixels 2.0 probes (acquired using [Open Ephys](https://open-ephys.org/)). Recordings are conducted
```